protoc-gen-go/grpc/grpc.go

```diff
@@ -303,6 +303,19 @@ func (g *grpc) generateClientMethod(servName, fullServName, serviceDescVar strin
 	if method.GetOptions().GetDeprecated() {
 		g.P(deprecationComment)
 	}
+	streamType := unexport(servName) + methName + "Client"
+	g.P(fmt.Sprintf(`// The ctx passed in will be used for the lifetime of this object.
```

Comment:
This is not a godoc-compliant comment.
```go
// <method> initiates a new streaming RPC to the server, yadda yadda.
```
However, the type that owns this method is not exported (`unexport(servName)` below).  This should be documented in the interface instead, or else it will never be seen.  Note that those include the comments from the proto file, so we'll have to be smart about how to keep pulling those in but also document the Go implementation as well.